apps/mobile/src/screens/connection.tsx

```diff
@@ -1,6 +1,7 @@
 import { RiseComponents } from '@rise-tools/kit'
```

Comment:
I guess we want to eventually remove all of these `kit` components because this represents the ones that are not organized in other packages. One day this would become the umbrella package

Reply:
Yup! Once we refactor dropdown (discussed in Discord) to a separate package with some better customization, this can be removed and replaced with umbrella!